filewriter_v2/README.md

```diff
@@ -0,0 +1,387 @@
+| [start_time](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-field) | NX_NUMBER[] | Dimension scale defining the axis `start_time` of the data. These are the relative start times for all images with absolute reference `/entry/start_time`. |
+| [channel](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-field) | NX_CHAR[][^2] | Dimension scale defining the axis `channel` of the data. These are the enabled channels, for example `["threshold_1", "threshold_2", "difference"]`. |
+| [data](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-data-data-field) | NX_NUMBER[nP,ch,i,j] | The image data.<br> **Note:** Files in V2 format always have only one `"/entry/data/data"` dataset, even in case `nimages_per_file` is greater than `0`. This is because it is a [Virtual Dataset](https://portal.hdfgroup.org/display/HDF5/Virtual+Dataset++-+VDS). |
+
+## /entry/instrument
+
+Collection of the components of the instrument or beamline.
+
+| Name | NeXus Type | Description |
+|---|---|---|
+| @NX_class | NX_CHAR | [`"NXinstrument"`](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-group) |
+| [name](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-name-field) | NX_CHAR | Name of the instrument. |
+| [time_zone](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-time-zone-field) | NX_DATE_TIME | ISO 8601 time_zone offset from UTC. |
+
+## /entry/instrument/beam
+
+Properties of the neutron or X-ray beam.
+
+| Name | NeXus Type | Description |
+|---|---|---|
+| @NX_class | NX_CHAR | [`"NXbeam"`](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-beam-group) |
+| [incident_energy](https://manual.nexusformat.org/classes/base_classes/NXbeam.html?highlight=nxbeam#nxbeam-incident-energy-field) | NX_FLOAT | Energy carried by each particle of the beam on entering the beamline component. |
+| incident_energy@units | NX_ENERGY | `"eV"` |
+| [incident_wavelength](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-beam-incident-wavelength-field) | NX_FLOAT | Wavelength of the beam in the case of monochromatic beam. |
+| incident_wavelength@units | NX_WAVELENGTH | `"angstrom"` |
+| [total_flux](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-beam-total-flux-field) | NX_FLOAT | Flux incident on beam plane in photons per second. In other words, this is the flux integrated over the area. In the case of a beam that varies in flux shot-to-shot, this is an array of values, one for each recorded shot.|
+| total_flux@units | NX_FREQUENCY | `"Hz"` |
+
+## /entry/instrument/detector
+
+| Name | NeXus Type | Description |
+|---|---|---|
+| @NX_class | NX_CHAR | [`"NXdetector"`](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-detector-group) |
```

Comment:
Link directly to https://manual.nexusformat.org/classes/base_classes/NXdetector.html#nxdetector